packages/core/src/sequelize.js

```diff
@@ -723,7 +724,7 @@ Use Sequelize#query if you wish to use replacements.`);
         await this.hooks.runAsync('beforeQuery', options, query);
         checkTransaction();
 
-        return await query.run(sql, bindParameters);
+        return await query.run(sql, bindParameters, { shouldMinifyAlias });
```

Comment:
Let's stay consistent in our naming

```suggestion
        return await query.run(sql, bindParameters, { minifyAliases });
```